public/javascripts/Gallery/src/displays/SeverityDisplay.js

```diff
@@ -26,36 +32,62 @@ function SeverityDisplay(container, severity, isModal=false) {
         if (isModal) {
             // Add bold weight. Find better way to do this.
             title.classList.add('modal-severity-header');
+            // centers tooltip
+            holder.classList.add('modal-no-severity-box')
```

Comment:
It looks like this CSS class isn't specific to there being no severity (even though that's what's motivating the change), so maybe change the name to something that isn't specific to a lack of severity?